### DISCUSSION TITLE
Antivirus Software blocking exe

### DISCUSSION THREAD

**Discussion by mrposch** [pyinstaller/pyinstaller#5407] [Help] [answered]
Hello,

I'm using conda and have issues when trying to use pyinstaller from a new virtual environment. When I use pyinstaller through my base environment everything works fine, but when use it within a virtual environment my company's security software will prevent the final exe file from running. I've tried using the same version numbers and still have the issue. Any ideas on how to solve this?

**Comment by bwoodsend** [comment, marked as answer]
Codesigning it might help - particularly if you're on macOS. And some systems don't like `--onefile` apps. But there's really not much we can do about this. There's no magic *I'm not a virus* setting you or PyInstaller can tick. Report it to your company's security software - it's their bug.

**Reply by mrposch** [reply]
I'm running on windows. I understand it may be the an issue in the security software, but if it works on my same system using the base conda environment maybe there's a way it's compiling differently between the environments. I'm not an expert so I don't know how to look into that but maybe there's a way to configure pyinstaller to do what it does in my base environment?

**Reply by mrposch** [reply]
I was able to figure this out. The new environment has python 3.8 and that was the issue. When I changed to python 3.7 as is in the base environment everything worked fine without the antivirus issues.
